test/e2e/resources/table.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: dynamodb.services.k8s.aws/v1alpha1
```

Comment:
please name this file a little more descriptively than just `table.yaml` :) Maybe something like `table_forums.yaml`?